jupyter/minimal/ubi8-python-3.8/requirements.in

```diff
@@ -1 +1,60 @@
+# Minimal notebook requirements
 jupyterlab==3.2.4
+jupyter_server==1.4.1
+notebook==6.4.12
+setuptools==65.4.0
+pip==22.2.2
+
+# JupyterLab extensions
+# A JupyterLab extension for version control using git
+jupyterlab-git==0.39.2
+# Jupyter Kernel Gateway is a web server that supports different mechanisms for spawning and communicating with Jupyter kernels.
+jupyter_kernel_gateway==2.4.0
```

Comment:
We can remove this extension since we are not using the command `jupyter kernelgateway`.

See https://jupyter-kernel-gateway.readthedocs.io/en/latest/getting-started.html#running